### ISSUE TITLE
Shape Tracing module not working with new fields types

### ISSUE THREAD

**Issue by bleroy** [bleroy/OrchardImportTest#2407]
Shape Tracing Module shows nothing when clicking on the new fields types
introduced in 1.4  

Work Item Details
--------------------
**Original CodePlex Issue:**	[Issue 18517](http://orchard.codeplex.com/workitem/18517)
**Status:**	Closed
**Reason Closed:**	Unassigned
**Assigned to:**	Unassigned
**Reported on:**	Mar 3, 2012 at 3:46 PM
**Reported by:**	Phusion
**Updated on:**	Mar 31, 2012 at 6:44 AM
**Updated by:**	sebastienros
**Closed on:**	Mar 31, 2012 at 6:44 AM
**Closed by:**	sebastienros
**Status:**	Closed
**Reason Closed:**	Unassigned
**Assigned to:**	Unassigned
**Reported on:**	Mar 3, 2012 at 3:46 PM
**Reported by:**	Phusion
**Updated on:**	Mar 31, 2012 at 6:44 AM
**Updated by:**	sebastienros
**Closed on:**	Mar 31, 2012 at 6:44 AM
**Closed by:**	sebastienros

**Comment by bleroy** [comment]
On *2012-03-13 06:10:51 UTC*, **@sebastienros** commented:

I tried with a NumericField and it works. I can browse everything from the Settings to the Field properties.
Can you provide a repro ?

**Comment by bleroy** [comment]
On *2012-03-13 20:33:32 UTC*, **@Phusion** commented:

Hi Sebastien, I will try to retrace the steps I took to have that issue and reproduce it. (when I get home :). thx

**Comment by bleroy** [comment]
On *2012-03-14 11:01:49 UTC*, **@Phusion** commented:

I have a custom content type, with lots of fields attached. For some reason, using the shape tracing I'm not getting any properties for this content type. Obviously the problem is on my side, because tonight I tried with a fresh instance of orchard and
 a simple content type I created for testing, and 4 different field types, the shape tracing worked fine. Does styling affect in anyway the tracing module?
